### DISCUSSION TITLE
ScanIndexForward true / false equivalent with OneTable?

### DISCUSSION THREAD

**Discussion by AndrewCathcart** [sensedeep/dynamodb-onetable#175] [Q&A] [answered]
Just curious how to go about ordering results returned from a query? Can't find anything about it in the docs.

Cheers

**Comment by mobsense** [comment, marked as answer]
Params.reverse:

| reverse | `boolean` | Set to true to reverse the order of items returned.|

**Reply by AndrewCathcart** [reply]
Ahh sorry missed that one - perfect. Does the library do any ordering at all on the results? Unsure how to e.g. order all users alphabetically or something like that, or order something by date. Would I need to do that on the dataset after the query returned?

**Reply by mobsense** [reply]
The ordering is done 100% by DynamoDB based on the `reverse` value.

Yes, you need to do your own custom ordering. We could in theory provide a sortByColumn utility which is what our SenseDeep app does.
